### PR TITLE
Add NextcloudPi to automated install options

### PR DESCRIPTION
**Pull request by p-bo**
Because they also provide install script - see Debian tab under Supported Systems section on nextcloudpi.com
obrazek

Signed-off-by: p-bo <pavel.borecki@gmail.com>

### CODE REVIEW

**Review by tflidd — APPROVED** [review]

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ archive <https://nextcloud.com/install/>`_. This method is recommended to instal
...
-install the community `Snap Package <https://snapcraft.io/nextcloud>`_. This includes a full production-ready stack, will maintain your HTTPS certificates for you, and will automatically update as needed to stay secure. You can also use the `Nextcloud VM scripts <https://github.com/nextcloud/vm/>`_ to install directly on a clean Ubuntu Server. It will setup everything for you and include scripts for automated installation of apps like; Collabora, OnlyOffice, Talk and so on. Please note that those two options are not officially supported by Nextcloud GmbH.
+install the community `Snap Package <https://snapcraft.io/nextcloud>`_. This includes a full production-ready stack, will maintain your HTTPS certificates for you, and will automatically update as needed to stay secure. You can also use the `Nextcloud VM scripts <https://github.com/nextcloud/vm/>`_ to install directly on a clean Ubuntu Server or `NextcloudPi scripts <https://nextcloudpi.com/>`_ (dtto atop Debian). It will setup everything for you and include scripts for automated installation of apps like; Collabora, OnlyOffice, Talk and so on. Please note that those three options are not officially supported by Nextcloud GmbH.
```
Comment (kesselb) [inline admin_manual/installation/source_installation.rst:17]: dtto atop Debian? :thinking:

Comment (p-bo) [reply admin_manual/installation/source_installation.rst:17]: Tipo (sorry for that) - should be ditto (https://en.wikipedia.org/wiki/Ditto) and "atop" as used for example here https://www.phoronix.com/news/Linux-Mint-LMDE5-Released. So in meaning "basically very similar thing, but based on Debian".

Comment (kesselb) [reply admin_manual/installation/source_installation.rst:17]: OK. Would you mind sending another pull request to fix the typo?

Comment (p-bo) [reply admin_manual/installation/source_installation.rst:17]: Yes, for sure - see https://github.com/nextcloud/documentation/pull/9473

**Pull request merged**